packages/slice-machine/components/Simulator/components/Header/index.tsx

```diff
@@ -57,6 +57,17 @@ const redirect = (
   void router.push(params.href, params.as, params.options);
 };
 
+const deviceToWidth = (device: Size) => {
+  switch (device) {
+    case Size.FULL:
+      return 1200;
```

Comment:
Where do these number com from? Could we sync them better with what we actually show the user (https://github.com/prismicio/slice-machine/blob/master/packages/slice-machine/components/Simulator/components/ScreenSizes/index.tsx#L31)?

Reply:
Also the iframe height is also fixed. Is fixing the width only enough?

Reply:
@mathieud fixing the height of the viewport doesn't make a difference since puppeteer will "scroll" to take a screenshot of the whole component. I've asked Amaury if this behaviour is expected or whether puppeteer should be configured to fix height